test/test_suit/test_schema_evolution_drop_table.py

```diff
@@ -46,7 +46,7 @@ def send(self):
         self.driver.sendBytesData(self.topic, value, key)
 
         # Sleep for some time and then verify the rows are ingested
-        sleep(60)
+        sleep(120)
```

Comment:
Curious why it's taking more time. I see you have tried multiple times.

Reply:
^ as well as why we're removing the 30 sec sleep below

Reply:
good question, this solution is not perfect, if we don't wait for more than 60 seconds than the in-memory offset is not updated and we will restart from the beginning, I put 120 seconds to make sure it's not flaky